rplugin/python3/LanguageClient/LanguageClient.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, nvim):
         self.diagnostics = {}
         self.lastLine = -1
         self.hlsid = None
+        self.insertUseName = ""
```

Comment:
The callback needed to know the name of the class in order to filter the results - so had to add this hack - don't know if there is a way to avoid this.